docs/migration/index.md

```diff
@@ -290,3 +290,43 @@ If you use snapshot testing and comment nodes are leaking into your snapshots, n
+| provide          | nested in `global`                                                                |
+| mixins           | nested in `global`                                                                |
+| plugins          | nested in `global`                                                                |
+| component        | nested in `global`                                                                |
+| directives       | nested in `global`                                                                |
+| attachToDocument | renamed `attachTo`. See [here](https://github.com/vuejs/vue-test-utils/pull/1492) |
+| scopedSlots      | removed. ScopedSlots are merged with `slots` in Vue 3                             |
+| context          | removed. Different from Vue 2, does not make sense anymore.                       |
+| localVue         | removed. No longer required - Vue 3 there is no global Vue instance to mutate.    |
+| listeners        | removed. No longer exists in Vue 3                                                |
+| parentComponent  | removed                                                                           |
+
+### Wrapper API (mount)
+
+| method         | notes                                                                                       |
+|----------------|---------------------------------------------------------------------------------------------|
+| find           | only `querySelector` syntax is supported. Use `findComponent(Comp)`                         |
+| setValue       | works for select, checkbox, radio button, input, textarea. Returns `nextTick`.              |
+| trigger        | returns `nextTick`. You can do `await wrapper.find('button').trigger('click')`              |
+| destroy        | renamed to `unmount` to match Vue 3 lifecycle hook name.                                    |
+| contains       | removed. Use `find`                                                                         |
+| emittedByOrder | removed. Use `emitted`                                                                      |
+| setSelected    | removed. Now part of `setValue`                                                             |
+| setChecked     | removed. Now part of `setValue`                                                             |
+| is             | removed                                                                                     |
+| isEmpty        | removed. use matchers such as [this](https://github.com/testing-library/jest-dom#tobeempty) |
+| isVueInstance  | removed                                                                                     |
+| name           | removed                                                                                     |
+| setMethods     | removed                                                                                     |
```

Comment:
I think this was removed in VTU v1.0.0.

Reply:
It still exists in the V1 docs with deprecation warning https://v1.test-utils.vuejs.org/api/wrapper/#setmethods. I will keep it in there